clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/network/view/AzureNetworkProvider.groovy

```diff
@@ -52,6 +57,17 @@ class AzureNetworkProvider implements NetworkProvider<AzureNetwork> {
     cacheView.getAll(Keys.Namespace.AZURE_NETWORKS.ns, RelationshipCacheFilter.none()).collect(this.&fromCacheData)
   }
 
+  @RequestMapping(value = "/azure/vnet/{account}/{region}/{name:.+}", method = RequestMethod.GET)
```

Comment:
Why are we exposing endpoints in the provider class and not through a controller? Who is expected to be calling this outside the two methods in cloud driver?